samples/CTFChallenges/readme.md

```diff
@@ -0,0 +1,27 @@
+# Purpose of this extension
+
+**This extension is meant for demo and training purposes.**
+
+The extension implements different scenarios with bad-performing AL Code. Some examples:
+ - Long-running queries
+ - Running substantial code in OnAfterGetRecord
+ - Listening for too many database triggers
+
+You can run these scenarios and use the standard troubleshooting tools to identify the root cause.
+
+The extension also contains a set of capture the flag debugging/snapshot debugging challenges.
+
+
+In addition to getting some hands-on experience with these tools, the examples also make it easy to _demonstrate_ to others how the tools work.
+
+# Installing the extension
+
+Build the extension (e.g. against your Business Central Online sandbox environment).
+
+Then upload the extension, either using the Extension Management page, or just publish the extension directly from VS Code.
+
+# Running the scenarios
+
+Use global search (alt-q) and search for the "Slow Code Examples" page (page 50100).
```

Comment:
The page to open would be named "CTF Challenges". Also, maybe mention how to run this without CTF portal mode.

Reply:
Hi Achim, thanks for the feedback! I have updated the files.